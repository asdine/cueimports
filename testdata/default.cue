package file

default: {
	color: colors.#Red
}
