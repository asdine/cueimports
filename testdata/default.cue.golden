package file

import "colors"

default: {
	color: colors.#Red
}
